lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -29,8 +29,7 @@ class CSSUsage extends Gatherer {
       .catch(err => {
         // TODO(phulce): Remove this once CSS usage hits stable
         if (/startRuleUsageTracking/.test(err.message)) {
-          this.failure = 'CSS Usage tracking requires Chrome \u2265 56';
-          return;
+          throw new Error('CSS Usage tracking requires Chrome \u2265 56');
```

Comment:
@patrickhulce now that errors are non-fatal by default, `afterPass` still throws but this error wins